beater/config/config.go

```diff
@@ -123,10 +123,10 @@ func DefaultConfig() *Config {
 		MaxHeaderSize:   1 * 1024 * 1024, // 1mb
 		MaxConnections:  0,               // unlimited
 		IdleTimeout:     45 * time.Second,
-		ReadTimeout:     30 * time.Second,
+		ReadTimeout:     3600 * time.Second,
```

Comment:
This is what I'm seeing in https://github.com/elastic/cloud-assets/blob/master/stackpack/apm/config/apm-server.yml#L77, but this seems pretty excessive. Any opinions?

Reply:
I think we should leave this one. Its value depends on the deployment of apm-server. In the Cloud case, there's a proxy which impose its own timeout.